src/main/kotlin/lotto/domain/TicketGenerator.kt

```diff
@@ -2,18 +2,19 @@ package lotto.domain
 
 class TicketGenerator(private val numberStrategy: NumberStrategy) {
 
-    private val TICKET_PRICE = 1000
-
-    fun tickets(amount: Int): List<Ticket> {
+    fun tickets(amount: Int): Tickets {
         val count = amount / TICKET_PRICE
-        return MutableList(count) {
+        return Tickets(MutableList(count) {
             this.ticket()
-        }
+        })
     }
 
     private fun ticket(): Ticket {
-        val pickedNumbers = numberStrategy.numbers()
-        return Ticket(pickedNumbers)
+        return Ticket(numberStrategy.numbers())
+    }
+
+    companion object {
+        val TICKET_PRICE = 1000
```

Comment:
여기서만 사용한다면 private로!
```suggestion
         const val TICKET_PRICE = 1000
```

Reply:
넵 반영했습니다. 감사합니다.